content/en/docs/reference/capabilities/_index.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Capabilities
+description: Capabilities required to run Falco
+weight: 20
+---
+
+To analyze kernel events we have [3 possible sources](/docs/event-sources/kernel/): kernel module, eBPF probe, modern eBPF probe.
+
+The kernel module can run only in privileged mode while the 2 eBPF probes could also use capabilities to run with fewer privileges.
+
+## eBPF probe
+
+The minimal set of capabilities required by Falco to run the eBPF probe is the following:
```

Comment:
```suggestion
The minimal set of capabilities required by Falco to run the eBPF probe are the following:
```

Reply:
Again, calling it "the eBPF probe" leaves me in state where I don't know which eBPF probe we are talking about.

Reply:
Here i left "is" since the subset should be "set"